### DISCUSSION TITLE
What is the correct way to setup CMD+S to save files on a mac

### DISCUSSION THREAD

**Discussion by thesobercoder** [LazyVim/LazyVim#5026] [Q&A]
Like the title says, I want to configure `CMD+S` to save files instead of `CTRL+S` because I have programmed my keyboard to swap `CTRL` with `CMD`.

Note: I use WezTerm

**Comment by HotThoughts** [comment]
I use. Wonder others approaches. 🤔
```
vim.keymap.set({ "i", "x", "n", "s" }, "<D-s>", "<cmd>w<cr><esc>", { desc = "Save file" })
```
